proj/src/main/webapp/notifications.js

```diff
@@ -101,14 +101,21 @@ function sendNotificationForChat(message) {
     const tag = sessionStorage.activeChatTag;
     const chatId = sessionStorage.activeChatId;
     const name = message.senderDisplay;
-    notificationBody = name+' has sent a message:\n\n'+message.content
+    notificationBody = name+' has sent a message:\n\n'+message.content;
+    var url;
+    if(window.location.href.includes("https://8080")){
+        url = 'https://8080-9f73b32e-6ccb-426e-8d6e-c03d6f324666.us-west1.cloudshell.dev/static/chat.html';
```

Comment:
why not just reroute to window.location.href? Won't this mean other people will reroute to your dev server?